cloud/kubernetes/helm/values.yaml

```diff
@@ -57,7 +61,7 @@ mysql:
   name: mysql
   containers:
     name: mysql
-    image: mysql:5.6
+    image: mysql:5.7
```

Comment:
As per discussion in our meeting, you mentioned that it has been verified in helm. So, I am giving approval.